docs/release-notes/release-notes-0.17.0.md

```diff
@@ -39,6 +39,8 @@ package](https://github.com/lightningnetwork/lnd/pull/7356)
 * [HTLC serialization updated](https://github.com/lightningnetwork/lnd/pull/7710) 
   to allow storing extra data transmitted in TLVs.
 
+* [Change LocalMaxCSVDelay Default](https://github.com/lightningnetwork/lnd/pull/7780).
```

Comment:
I think it's called `MaxLocalCSVDelay` and we should mention `--maxlocaldelay` instead, plus if users want a different value they can set it themselves. Just in case someone relies on the old default(hopefully not...)

Reply:
sounds good